Makefile

```diff
@@ -1,30 +1,43 @@
-SHELL := /usr/bin/env bash
+include boilerplate/generated-includes.mk
 
-# Include shared Makefiles
-include project.mk
-include standard.mk
-include functions.mk
+SHELL := /usr/bin/env bash
 
-default: generate-syncset gobuild
+default: generate-syncset go-build
```

Comment:
This is overwriting the `default` target set deep in the included makefiles.

Reply:
Thanks, I've now removed this.